src/components/ScriptExplorer/OutputEntry.jsx

```diff
@@ -43,7 +43,7 @@ class OutputEntry extends React.Component {
       multisig: PropTypes.shape({
         address: PropTypes.string,
       }),
-    }).isRequired,
```

Comment:
`changeNode` isn't required because there is no change output on the script explorer.